docs/UserGuide.md

```diff
@@ -140,33 +144,33 @@ Wishlist | TravelPlan
 --------------------------------------------------------------------------------------------------------------------
 ## Parameters
 
-The table below shows the type of parameters each different object requires.
+The tables below show the different type of parameters each object requires.
 Do note that all parameters stated are compulsory when creating the object.
 Declaration of the parameters strictly follows the format given.
 
 #### Activity Parameters
 
-Name of Parameter | Description | Requirement
+Name of Parameter | Description | Format
 ------------ | ------------- | -------------
-`n/NAME` | Name of the activity. | Names should only contain alphanumeric characters, punctuations and spaces, and it should not be blank.
-`l/LOCATION` | Location/ Address of the activity | Location can take any values, and it should not be blank.
-`i/LEVEL_OF_IMPORTANCE` | The priority assigned to the activity | Importance Level should only contain numbers, and it should range from 1 - 5, with 5 being the most important while 1 being the least important.
-`c/COST` | Cost of the activity | Cost should only contain numbers, and it should be a positive integer.
-`d/DATE_AND_TIME` | Date and Time intended to do the activity | Format of date is in YYYY-MM-DD and format of time is HH:MM (24h clock). 
+`n/NAME` | Name of the activity. | <ul><li>Should only contain alphanumeric characters, punctuations and spaces</li><li>Should not be blank</li></ul>
+`l/LOCATION` | Location/ Address of the activity | <ul><li>Can take any values</li><li>Should not be blank</li></ul>
+`i/LEVEL_OF_IMPORTANCE` | The priority assigned to the activity | <ul><li>Should only contain numbers</li><li>Should range from 1 - 5 <br/>(most important = 5, least important = 1)</li></ul>
+`c/COST` | Cost of the activity | <ul><li>Should only contain numbers</li><li>Should be a positive integer</li></ul>
```

Comment:
Can change this to `Should only contain positive integer not exceeding MAX_INT`?